src/milia/api/io.cljs

```diff
@@ -46,7 +46,7 @@
   Authorization, X-CSRFToken and Cache-control headers where necessary"
   [& {:keys [get-crsftoken? must-revalidate? accept-header]}]
   (let [temp-token (:temp-token @*credentials*)]
-    (into {} [(when (and temp-token (is-not-null? temp-token))
+    (into {} [(when (and (not-empty temp-token) (is-not-null? temp-token))
```

Comment:
`not-empty` works the same way as `not-empty? `
`not-empty?` was our custom version https://github.com/onaio/zebra/blob/eb5509fc57d35221f0a8e30383eadf680ffff095/src/clj/ona/utils/string.cljc#L14